sdv/single_table/base.py

```diff
@@ -134,6 +134,10 @@ def set_address_columns(self, column_names, anonymization_level='full'):
             anonymization_level (str):
                 The anonymization level to use for the address transformer.
         """
+        warnings.warn(
+            '`set_address_columns` is deprecated. Please add these columns directly to your'
+            ' metadata using `add_column_relationship`.', DeprecationWarning
+        )
```

Comment:
I think we said we can stop doing the logic in this method and just make it a no-op with a warning